src/openvslam/tracking_module.cc

```diff
@@ -115,6 +119,10 @@ Mat44_t tracking_module::track_stereo_image(const cv::Mat& left_img_rect, const
     const auto end = std::chrono::system_clock::now();
     elapsed_ms_ = std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count();
 
+    if (!tracking_started_) {
```

Comment:
I'll admit I'm not fully tracking (ha...ha...) the use of `tracking_started_`. If this wasn't needed before for localization, I'm not entirely sure why its needed now?